testkit/build.rs

```diff
@@ -0,0 +1,32 @@
+extern crate exonum_build;
+
+use exonum_build::protobuf_generate;
+
+fn main() {
+    protobuf_generate("src/proto", &["src/proto"], "testkit_protobuf_mod.rs");
+
+    let exonum_protos = exonum_build::get_exonum_protobuf_files_path();
+    protobuf_generate(
+        "tests/inflating_currency/proto",
+        &["tests/inflating_currency/proto", &exonum_protos],
+        "currency_example_protobuf_mod.rs",
+    );
+
+    protobuf_generate(
+        "tests/counter/proto",
+        &["tests/counter/proto"],
+        "counter_example_protobuf_mod.rs",
+    );
+
+    protobuf_generate(
+        "tests/service_hooks/proto",
+        &["tests/service_hooks/proto"],
+        "hooks_example_protobuf_mod.rs",
+    );
+
+    protobuf_generate(
+        "examples/timestamping/proto",
+        &["examples/timestamping/proto"],
+        "timestamping_example_protobuf_mod.rs",
+    );
+}
```

Comment:
Or even to create some structure in `exonum_build` crate for holding this data:
```rust
struct ProtoData {
    proto_path: String,
    includes: Vec<String>,
   mod_file_name: String
}
```

Reply:
I think they are equivalent because there are no reduction in the code size.
To me current version looks more simple.

Reply:
Really ? What about DRY ?

Reply:
I'd support @dvush in that repeated function calls and repeated tuples/structures with a loop (or `map`) have the same readability. We have to repeat either function calls or struct/tuple constructors so it does not make any difference here, so I'm personally fine with function calls. Verbosity here comes from essential complexity of having 4 sets of 3 distinct paths that are not going anywhere.

There may be other considerations for choosing structs (and _not_ tuples) because with something like this

```rust
Proto {
    proto_dir: "tests/service_hooks/proto",
    include_dirs: vec!["tests/service_hooks/proto"],
    rust_module: "hooks_example_protobuf_mod.rs",
}
```

it's easier to see what path stands for what, when compared to

```rust
(
    "tests/service_hooks/proto",
    vec!["tests/service_hooks/proto"],
    "hooks_example_protobuf_mod.rs",
)
```

Such 'named arguments' may actually improve readability.